### ISSUE TITLE
Support Micronaut

### ISSUE THREAD

**Issue by ralscha** [cettia/asity#46]
[Micronaut](https://micronaut.io/)

Micronaut features a Dependency Injection and Aspect-Oriented Programming runtime that uses no reflection. It runs on Netty.

**Comment by flowersinthesand** [comment]
Looks great.

**Comment by ralscha** [comment]
Indeed. Supports everything Spring provides but without any reflection. Instead it uses Annotation Processors. Results in faster startup time and less memory usage. 

There is a similar project from RedHad: [Quarkus](https://quarkus.io/)
I think this also uses Netty under the hood.
